app/js/lib/form_components.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+var objectAssign = require( 'object-assign' ),
+	actions = require( './actions' ),
+	createDefaultChangeHandler = function ( store, contentName ) {
+		return function ( evt ) {
+			store.dispatch( actions.newChangeContentAction( contentName, evt.target.value ) );
+		};
+	},
+	RadioComponent = {
+		element: null,
+		contentName: '',
+		onChange: null,
+		render: function ( formContent ) {
```

Comment:
Do you mean `formData` instead of `formContent`?

I was talking about the method name. I think `formContent` is clear enough.